### PR TITLE
feat!: migrate to new slider, fix new slider styles

### PR DESCRIPTION
**Pull request by qradle-yndx**

### CODE REVIEW

File: src/blocks/HeaderSlider/HeaderSlider.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {MobileContext} from '../../context/mobileContext';
 import {HeaderSliderBlockProps, SliderType} from '../../models';
 import {block} from '../../utils';
 import Header from '../Header/Header';
-import {SliderBlock} from '../index';
+import {SliderNewBlock as SliderBlock} from '../unstable';
```
Comment (NikitaCG) [inline src/blocks/HeaderSlider/HeaderSlider.tsx:7]: if it is a breaking change I think we need to delete old slider and use new like as the main one

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/blocks/HeaderSlider/HeaderSlider.tsx (hunk 1 above)
Comment (qradle-yndx) [reply src/blocks/HeaderSlider/HeaderSlider.tsx]: I replaced SliderBlock with the new one and renamed old one as SliderOldBlock

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gravity-ui/page-constructor",
-  "version": "5.28.7",
+  "version": "5.27.1-alpha.2",
```
Comment (aeksandla) [inline package.json:3]: This should be deleted

File: package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@gravity-ui/page-constructor",
-  "version": "5.28.7",
+  "version": "5.27.1-alpha.2",
```
Comment (aeksandla) [inline package-lock.json:3]: This should be deleted

**Review by aeksandla — CHANGES_REQUESTED** [review]

File: src/blocks/Slider/__stories__/Slider.stories.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ import Slider from '../Slider';
 import data from './data.json';
 
 export default {
-    title: 'Blocks/Slider',
+    title: 'blocks/Slider',
```
Comment (NikitaCG) [inline src/blocks/Slider/__stories__/Slider.stories.tsx:12]: looks like unnecessary diff

File: src/blocks/Slider/__stories__/data.json [hunk 5]
```diff
@@ -107,7 +107,7 @@
     "content": {
       "type": "slider-block",
       "title": {
-        "text": "Slider",
+        "text": "Slider New",
```
Comment (NikitaCG) [inline src/blocks/Slider/__stories__/data.json:110]: I think we can delete `New`

File: src/blocks/SliderOld/Arrow/Arrow.tsx [hunk 6]
```diff
@@ -7,24 +7,22 @@ import {i18n} from '../i18n';
...
-    extraProps?: React.ButtonHTMLAttributes<HTMLButtonElement>;
 }
 
-const Arrow = ({type, onClick, className, size = 16, extraProps}: ArrowProps & ClassNameProps) => (
+const Arrow = ({type, handleClick, className, size = 16}: ArrowProps & ClassNameProps) => (
     <div className={b({type}, className)}>
         <button
             className={b('button')}
-            onClick={onClick}
+            onClick={() => handleClick && handleClick(type)}
```
Comment (NikitaCG) [inline src/blocks/SliderOld/Arrow/Arrow.tsx:24]: maybe `handleClick ? () => handleClick() : null`?

File: src/blocks/SliderOld/Arrow/Arrow.scss [hunk 7]
```diff
@@ -23,6 +23,7 @@ $block: '.#{$ns}slider-new-block-arrow';
         #{$root}__icon-wrapper {
             transform: rotate(180deg);
         }
+        margin-right: 16px;
```
Comment (NikitaCG) [inline src/blocks/SliderOld/Arrow/Arrow.scss:26]: use `$indent...`

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: styles/variables.scss [hunk 8]
```diff
@@ -57,3 +57,4 @@ $gridGutterMobile: 8px;
 $contentWidth: calc(#{$newContentWidth} + #{$gridGutter} * 2 + #{$gridContainerPadding} * 2);
 
 $gridContainerMargin: 16px;
+$fsImageMobilePadding: 8px;
```
Comment (aeksandla) [inline styles/variables.scss:60]: what is 'fs'?

Comment (qradle-yndx) [reply styles/variables.scss:60]: fullscreen - should i rename it?

Comment (aeksandla) [reply styles/variables.scss:60]: yes, lets rename it

File: src/blocks/Slider/__stories__/data.json (hunk 5 above)
Comment (qradle-yndx) [reply src/blocks/Slider/__stories__/data.json]: fixed. Also screenshot test images were updated

File: src/blocks/SliderOld/Arrow/Arrow.tsx (hunk 6 above)
Comment (qradle-yndx) [reply src/blocks/SliderOld/Arrow/Arrow.tsx]: changed to `handleClick ? () => handleClick(type) : undefined`

**Review by aeksandla — APPROVED** [review]
